### PR TITLE
Add release notes for v3.0.0

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
@AdmiringWorm @JPRuskin can I also get some eyes on this PR as well, thanks!

**Review by JPRuskin — APPROVED** [review]
LGTM!

File: input/en-us/licensed-extension/release-notes.md [hunk 1]
```diff
@@ -26,6 +26,35 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
 
 > :memo: **NOTE** We've identified an issue with Self-Service "Interactive" and UAC - we are working on a fix. Please see [#36](https://github.com/chocolatey/chocolatey-licensed-issues/issues/36) and subscribe for details. Until then, do not turn on the interactive feature of self-service or nothing will work.
 
+## 3.0.0 (December 8, 2021)
+
+> :warning: **WARNING**
+>
+> The dependencies of the chocolatey.extension package have changed in this release, and it now requires Chocolatey CLI v0.11.1.
```
Comment (AdmiringWorm) [inline input/en-us/licensed-extension/release-notes.md:33]: This reads to me that it requires exactly version 0.11.1 (although I know better).

Should it maybe say `0.11.1 or higher`, or `0.11.1+`?
Not sure what the typical convention would be.

File: input/en-us/licensed-extension/release-notes.md [hunk 2]
```diff
@@ -26,6 +26,35 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
...
+ * Package Builder:
+   * Fix - Ensure "--version" option is ignored when using choco new command- see [licensed #203](https://github.com/chocolatey/chocolatey-licensed-issues/issues/203)
+   * Fix - Ensure checksums are calculated correctly when using both x86 and x64 urls that result in the same file name - see [licensed #94](https://github.com/chocolatey/chocolatey-licensed-issues/issues/94)
+ * Fix - Package Internalizer: Remove authenticode signatures when combining files - see [licensed #155](https://github.com/chocolatey/chocolatey-licensed-issues/issues/155)
+ * Fix - Package Builder UI: Ensure that long file paths don't wrap within input textboxes
+
+### IMPROVEMENTS
+ * Package Builder / Package Internalizer - Generated scripts should include validation of Chocolatey license - see [blog post](https://blog.chocolatey.org/2021/09/chocolatey-licensed-changes-restricted-to-licensed-nodes/) for additional information
+ * PowerShell:
+   * Ensure that all CmdLets are using consistent parameter names by using aliases
```
Comment (AdmiringWorm) [inline input/en-us/licensed-extension/release-notes.md:55]: hmm, first time I have seen `Cmdlets` being spelled with an upper case `L`. Should probably be
```suggestion
   * Ensure that all Cmdlets are using consistent parameter names by using aliases
```

File: input/en-us/licensed-extension/release-notes.md [hunk 3]
```diff
@@ -26,6 +26,35 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
...
+
+> :warning: **WARNING**
+>
+> The dependencies of the chocolatey.extension package have changed in this release, and it now requires Chocolatey CLI v0.11.1.
+
+### BREAKING CHANGES
+ * Package Builder / Package Internalizer:
+   * Generated packaging scripts will now only work with correctly licensed Chocolatey clients - see [blog post](https://blog.chocolatey.org/2021/09/chocolatey-licensed-changes-restricted-to-licensed-nodes/) for additional information
+ * The for-public option has been removed from the choco new command
+   * Given the [changes](https://blog.chocolatey.org/2021/09/chocolatey-licensed-changes-restricted-to-licensed-nodes/) that are included in this release, this option is no longer supported
```
Comment (AdmiringWorm) [inline input/en-us/licensed-extension/release-notes.md:39]: I didn't see it covered in the linked blog post, so should there be a mention here that public packages can be created when commercial arguments are **NOT** used?

Comment (gep13) [reply input/en-us/licensed-extension/release-notes.md:39]: Yes, that is perhaps a good idea, I will get that re-worded, just testing a couple things just now

File: input/en-us/licensed-extension/release-notes.md (hunk 1 above)
Comment (gep13) [reply input/en-us/licensed-extension/release-notes.md:33]: I will fix this up with a `+`.

**Comment by gep13** [comment]
@AdmiringWorm can you have another review here? Thanks!

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Pull request merged**